src/api/app/assets/javascripts/webui/application.js

```diff
@@ -60,7 +60,6 @@
 //= require webui/request_show_redesign/bs_request_actions.js
 //= require webui/request_show_redesign/rpm_lint_results.js
 //= require webui/delete_confirmation_dialog.js
-//= require webui/nav_tabs.js
```

Comment:
This js is available globally. We don't need it for separate actions. I've moved it to beta_show page